addons/website_event_track/models/event_sponsor.py

```diff
@@ -27,10 +27,11 @@ class Sponsor(models.Model):
     partner_email = fields.Char('Email', related='partner_id.email')
     partner_phone = fields.Char('Phone', related='partner_id.phone')
     partner_mobile = fields.Char('Mobile', related='partner_id.mobile')
-    url = fields.Char('Sponsor Website')
+    url = fields.Char('Sponsor Website', related='partner_id.website')
```

Comment:
Small note: making it a related unstore it (no column anymore in db). Either we don't need this field in db and we should probably write a small migration script to tell that the column should be removed (remove_column) . Either we want to keep this field and you have to make it stored (in which case it will not behave like other related fields). Probably to confirm with PO.

Reply:
I'd say if it's going to be like the rest of related fields and just grabbed from the partner's data, it wouldn't make much sense to keep this one stored in the database and not the rest, unless this field was used by some user to store a webpage different than the one in the partner.

But in that case, with the new requirements it would also be lost since it would be overwritten by the partner one with no ability to change, regardless of whether it is stored in the database or not. So in this case, maybe a better idea would be to leave it as stored, and instead of making the field related, just add the partner website as a suggestion?